include/envoy/http/filter.h

```diff
@@ -508,6 +523,14 @@ class StreamEncoderFilter : public StreamFilterBase {
    */
   virtual FilterTrailersStatus encodeTrailers(HeaderMap& trailers) PURE;
 
+  /**
+   * Called with metadata to be encoded. The function always returns continuing filter iteration.
```

Comment:
"The function always returns continuing filter iteration." -> @\return

Reply:
Fixed.